src/pages/iou/steps/MoneyRequstParticipantsPage/MoneyRequestParticipantsSelector.js

```diff
@@ -170,6 +170,7 @@ class MoneyRequestParticipantsSelector extends Component {
                 boldStyle
                 safeAreaPaddingBottomStyle={this.props.safeAreaPaddingBottomStyle}
                 shouldShowOptions={isOptionsDataReady}
+                shouldDelayFocus
```

Comment:
Do you mean that `shouldDelayFocus` is a hack in general, or just this specific application? I wasn't aware of the problem with skipped keystrokes you're describing.

Do you have any other ideas for fixing https://github.com/Expensify/App/issues/24314?

Reply:
Before the fix the text input didn't properly focus at all after the navigation, so how could you type fast and enter the mentioned keystrokes? Could you maybe provide a video?

Reply:
When was that recorded? Is this before/after applying changes from this PR?

From my perspective the case is quite pragmatic. We have an issue, https://github.com/Expensify/App/issues/24314. You can't type at all after the navigation, leave alone type fast. So from my perspective, the suggested solution greatly improved the situation if not completely fixed.

@luacmartins What do you think? It's a tricky case for me, as such debates typically take place at the proposal selection phase. We're currently at the lat stage of the implementation. Maybe it would be acceptable to push this to the users on 11th, so the bug that affects them is fixed, and than we can open another one for the missed keystrokes and prioritize it appropriately?

Reply:
> we can always update the solution

I wasn't aware of the alternative solution.

@Swor71 Could you check that you reproduce the skipped keystrokes problem on your PR branch? If so, I agree that that's a flaw that should be fixed. @rushatgabhane provided a suggestion.

> or go back to the issue and discuss proposals over there.

There was no proposal selection phase and no proposals.

Reply:
> There was no proposal selection phase and no proposals.

It's wrong step.
Even though agency member, there should be proposal review before raising PR

Reply:
@situchan if you missed the conversation in the [issue](https://github.com/Expensify/App/issues/24314), there was [this comment](https://github.com/Expensify/App/issues/24314#issuecomment-1688825796) from Carlos about no need for proposal, hence this situation.

@cubuspl42 @rushatgabhane 
Maybe it's just me but this keystroke 'problem' seems very forced and I totally agree with @cubuspl42 that if the input is not focused, why would we even discuss fast typing to begin with? 

It's obviously up to you how you'd like to proceed, but I'd lean towards @cubuspl42 suggestion about moving on with this as it actually solves the issue and then maybe revisit the keystroke discussion or a more generalised solution if this would be happening on other pages. Just my 2 cents.

Reply:
> @situchan if you missed the conversation in the https://github.com/Expensify/App/issues/24314, there was https://github.com/Expensify/App/issues/24314#issuecomment-1688825796 from Carlos about no need for proposal, hence this situation.

Gotcha. So it's edge case.

Here's general flow: https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#for-external-agencies-that-expensify-partners-with

Reply:
@Swor71 @cubuspl42 it seems like we've discussed this many times and we decided to go with using a ref and `onTransitionEnd` [here](https://expensify.slack.com/archives/C01GTK53T8Q/p1690301750089799?thread_ts=1689957344.255689&cid=C01GTK53T8Q). So I think we should be consistent and change the solution to that as well. Sorry about the back and forth here.

Reply:
@luacmartins I wasn't aware of the mentioned consensus (or at least I didn't remember). I'll have it in mind for the future.

@Swor71 Nothing more to be said; the PR should be updated with the `onTransitionEnd` approach. Also sorry for the complications.

Reply:
No worries, I was part of that thread and had also forgotten about it 😅